tests/energycal_test.py

```diff
@@ -70,6 +70,18 @@ def cal_spec(spec_data):
     return bq.Spectrum(spec_data, bin_edges_kev=TEST_EDGES_KEV)
 
 
+def linear_regression(x, y):
```

Comment:
Why not use [`numpy.polyfit(x, y, 1)`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.polyfit.html)?

Reply:
is used internally, I thought it would add some more confidence if another method produces the same result, but by now I tend to leave these tests out anyways, as it is covert sort of in 

 ```
def test_linear_fitting_simple():
    """Test fitting of known slope and offset"""

    cal = bq.LinearEnergyCal.from_points(chlist=[0.0, 1.0], kevlist=[1.0, 3.0])
    assert np.allclose([1.0, 2.0], [cal.offset, cal.slope])
```